src/test/resources/graphql/locations/location.graphql

```diff
@@ -0,0 +1,7 @@
+query LocationInfo($region: RegionType!) {
```

Comment:
Maybe rename this file to "getLocation".

Reply:
OK